package.json

```diff
@@ -195,6 +195,7 @@
     "core-js": "^3.9.1",
     "date-fns": "^2.15.0",
     "jszip": "^3.8.0",
+    "oembed-providers": "^1.0.20240528",
```

Comment:
needs a quick security review

Reply:
Yep! Opened one here already 🙂 https://github.com/brave/reviews/issues/1649

Reply:
it looks like we are not running any code from this npm package, just using a JSON file that it provides? can we just bundle https://oembed.com/providers.json and update it periodically as needed? that would avoid adding an additional dependency which could be compromised someday, add postinstall scripts, etc.

Reply:
We can certainly do that if we want. I mostly choose this route as it provided the easiest way to keep track of its updates, but yeah its just a single json file we're pulling in so if we want I can just drop the file into the iOS project instead.

Reply:
Pushed the change, providers json file is now just bundled with the app directly and no changes to package.json anymore!